gno.land/pkg/integration/testdata/object_pointer_2.txtar

```diff
@@ -0,0 +1,56 @@
+# Load the packages and start the node
```

Comment:
This txtar already works in master (as of 3-sep-2025) and gives the same results as in your branch. I do not understand the purpose of this specific test in the context of your changes of GetPackages, although the changes seem to be correct.

Reply:
I also noticed that on master, the execution of this txtar takes 2s, versus 3s on your branch (in `gno.lang/pkg/integration`: `go test -run=TestTestdata/object_pointer_2`).
Is this overhead of 1 second coming from getting package from store vs just inferring it from realm ?

Reply:
There also seems to be a 1 minute slowdown in general on the gno.land tests between here and master...